Rakefile

```diff
@@ -0,0 +1,11 @@
+begin
+  require "chefstyle"
+  require "rubocop/rake_task"
+  desc "Run Chefstyle tests"
+  RuboCop::RakeTask.new(:style) do |task|
+    task.options += ["--display-cop-names", "--no-color"]
+  end
+rescue LoadError
+  puts "chefstyle gem is not installed. bundle install first to make sure all dependencies are installed."
```

Comment:
I think traditionally we wanted to be able to exclude the stuff, but that probably doesn't make sense here. It all goes back to github changelog generator where we wanted to be able to skip that group but still run style w/o issues. It can probably go away at this point or yeah reraise.